tpe/lib/src/Link.cc

```diff
@@ -42,3 +42,44 @@ Entity &Link::AddCollision()
   this->ChildrenChanged();
   return *it->second.get();
 }
+
+//////////////////////////////////////////////////
+void Link::SetLinearVelocity(const math::Vector3d &_velocity)
+{
+  this->linearVelocity = _velocity;
+}
+
+//////////////////////////////////////////////////
+math::Vector3d Link::GetLinearVelocity() const
+{
+  return this->linearVelocity;
+}
+
+//////////////////////////////////////////////////
+void Link::SetAngularVelocity(const math::Vector3d &_velocity)
+{
+  this->angularVelocity = _velocity;
+}
+
+//////////////////////////////////////////////////
+math::Vector3d Link::GetAngularVelocity() const
+{
+  return this->angularVelocity;
+}
+
+//////////////////////////////////////////////////
+void Link::UpdatePose(
+  const double _timeStep,
+  const math::Vector3d &_linearVelocity,
+  const math::Vector3d &_angularVelocity)
```

Comment:
Is there a use case for calling this function with velocities different from the ones internal to the link? Shouldn't we be using `this->linearVelocity` and `this->angularVelocity` here instead of passing them in? Otherwise these values are never used by the link itself.

Reply:
don't think there is a use case for it so removed `_linearVelocity` and `_angularVelocity` args. 58f3e0b